apache/trafficcontrol
# TP: bug fixes on the any map form

## What does this PR (Pull Request) do?

- [x] This PR fixes #1943 and fixes #2426 and fixes #4102 and fixes #4201 


This PR actually fixes 4 issues. I could have done 4 PR's but figured since each is so small and related to the any map delivery service form, I'd just submit one PR.

This PR also makes "Raw Remap Text" a required field on the ANY_MAP DS form otherwise the DS cannot serve content.

- The Any Map ds form is not currently documented, therefore, making changes does not require a documentation change.
- The existing UI tests server as an adequate regression test of the any map delivery service form.

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Issue #1943:
- ensure that the Routing Configuration Settings section has been removed from the ANY_MAP delivery service form.

Issue #2426:
- ensure that the "Manage Invalidation Requests" menu item does not exist under More on the any map delivery service form.

Issue #4102 
- ensure that the delivery service URLs section has been removed from the ANY_MAP delivery service form. 

Issue #4201 :
- ensure that the "Manage Regexes" and "Manage Static DNS Entries" menu item does not exist under More on the any map delivery service form.

## If this is a bug fix, what versions of Traffic Control are affected?
- master (e3c48b6)

## The following criteria are ALL met by this PR
